scheduler-interface-spec.md

```diff
@@ -233,6 +230,9 @@ message RegisterResourceManagerRequest {
 
   // Pass the build information of k8shim to core.
   map<string, string> buildInfo = 4;
+
+  // Pass the serialized configuration for this policyGroup to core.
+  bytes config = 5;
```

Comment:
The yaml that we use for the config should never contain anything that cannot be represented as a normal UTF8 character. For compatibility we should use a simple `string` type. We do not want to support arbitrary binary values in the message.

Reply:
Updated.